dist/constant/message.js

```diff
@@ -27,6 +27,6 @@ exports.MESSAGES = {
     USE_WEBPACK_PROMPT: 'Do you want to use webpack?',
     WELCOME_MESSAGE: 'Welcome to kintone-cli!',
     WEBPACK_ENTRY_PROMPT: 'What is the entry for webpack?',
-    USE_PROXY_PROMT: 'Do you use proxy?'
+    USE_PROXY_PROMPT: 'Do you use proxy?'
```

Comment:
I mean that this is unnecessary because there is the same constant at line #23.
Please double-check it.

Reply:
Sorry I update wrong commit, I deleted it in here:
https://github.com/kintone-labs/kintone-cli/pull/106/commits/d8c5ce318ffbfccd8dbbb669ad18a5dd017bad76

Reply:
Got it. Ok now!